ansible/module_utils/port_utils.py

```diff
@@ -103,12 +103,14 @@ def get_port_alias_to_name_map(hwsku, asic_name=None):
                 or hwsku == "Arista-7050CX3-32S-C32":
             for i in range(1, 33):
                 port_alias_to_name_map["Ethernet%d/1" % i] = "Ethernet%d" % ((i - 1) * 4)
-        elif hwsku == "Arista-7060DX5-64S":
```

Comment:
@bingwang-ms Do you remove support for the Arista-7060DX5-64S fanout?

Reply:
Good catch! I added it back. Thanks!